utils/frame/rpc/system/src/lib.rs

```diff
@@ -89,13 +83,8 @@ pub struct FullSystem<P: TransactionPool, C, B> {
 
 impl<P: TransactionPool, C, B> FullSystem<P, C, B> {
 	/// Create new `FullSystem` given client and transaction pool.
-	pub fn new(client: Arc<C>, pool: Arc<P>, deny_unsafe: DenyUnsafe,) -> Self {
-		FullSystem {
-			client,
-			pool,
-			deny_unsafe,
-			_marker: Default::default(),
-		}
+	pub fn new(client: Arc<C>, pool: Arc<P>, deny_unsafe: DenyUnsafe) -> Self {
+		FullSystem { client, pool, deny_unsafe, _marker: Default::default() }
```

Comment:
I find this rather unfortunate formatting and harder to read than the original.

Reply:
Could try struct_lit_single_line=false ? ( https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#struct_lit_single_line )

Reply:
This happened when I tried that:

Pretty unfortunate 😬 
```diff
diff --git a/utils/fork-tree/src/lib.rs b/utils/fork-tree/src/lib.rs
index 3d5cc85722..0a140326fe 100644
--- a/utils/fork-tree/src/lib.rs
+++ b/utils/fork-tree/src/lib.rs
@@ -1195,10 +1219,42 @@ mod test {
 				}
 			};
 
-			tree.import("A0", 1, Change { effective: 5 }, &is_descendent_of).unwrap();
-			tree.import("A1", 1, Change { effective: 5 }, &is_descendent_of).unwrap();
-			tree.import("D", 10, Change { effective: 10 }, &is_descendent_of).unwrap();
-			tree.import("E", 15, Change { effective: 50 }, &is_descendent_of).unwrap();
+			tree.import(
+				"A0",
+				1,
+				Change {
+					effective: 5,
+				},
+				&is_descendent_of,
+			)
+			.unwrap();
+			tree.import(
+				"A1",
+				1,
+				Change {
+					effective: 5,
+				},
+				&is_descendent_of,
+			)
+			.unwrap();
+			tree.import(
+				"D",
+				10,
+				Change {
+					effective: 10,
+				},
+				&is_descendent_of,
+			)
+			.unwrap();
+			tree.import(
+				"E",
+				15,
+				Change {
+					effective: 50,
+				},
+				&is_descendent_of,
+			)
+			.unwrap();
 
 			(tree, is_descendent_of)
 		};
```

Reply:
Idk, having it all on one line seems fine as long as it's not overly long. When it does get too long it breaks it down to one arg per line. There's a break even point where one is preferable to the other - I think we just have to agree on max line length.

Reply:
In general, I am totally in favor of having (any) kind of standard that can be applied automatically. This is a time & brain saver.

The example above is probably very marginal but indeed annyoing. There are times where you have data (even long lines) and it is way more readable to keep it on single lines.

The issue above seem addressable using `struct_lit_single_line = true`.

Alternatively, I think this can also be shutdown for those exceptions with:
```
#[rustfmt::skip]
{
  tree.import("A0", 1, Change { effective: 5 }, &is_descendent_of).unwrap();
  tree.import("A1", 1, Change { effective: 5 }, &is_descendent_of).unwrap();
  tree.import("D", 10, Change { effective: 10 }, &is_descendent_of).unwrap();
  tree.import("E", 15, Change { effective: 50 }, &is_descendent_of).unwrap();
}
```

It works however it changes the meaning of the code.